Sources/FisherYates/random.swift

```diff
@@ -15,11 +15,14 @@
 #else
     import Glibc
 
+    private let seedRandom: Void = {
+        srandom(UInt32(time(nil)))
+    }()
+
     public let random: (Int) -> Int = {
         while true {
             let x = Glibc.random() % $0
-            let y = Glibc.random() % $0
-            guard x == y else { return x }
+            if x < (RAND_MAX - RAND_MAX % Int32($0)) { return x }
```

Comment:
I think this should be:

let upperLimit = RAND_MAX - RAND_MAX % Int32($0)
while true {
    let x = Glibc.random()
    if x < upperLimit { return x % $0 }
}

Reply:
D'oh! Yes, of course. Let me fix it.